jupyterlab/upgrade_extension.py

```diff
@@ -13,14 +13,16 @@
     raise RuntimeError("Please install cookiecutter")
 
 
-COOKIECUTTER_BRANCH = "3.0"
+DEFAULT_COOKIECUTTER_BRANCH = "3.0"
```

Comment:
Wondering whether this should default to the `master` branch of the cookiecutter, which targets the latest stable version of JupyterLab?

Staying on `3.0` sounds fine, but will require manually backporting iterations on the cookiecutter repo to its `3.0` branch.

Reply:
I was hesitating to switch to master too. But I think pinning the version is better to ensure consistency for the user. If it run `jupyterlab.upgrade_extension` script from JupyterLab 3, it should not see upgrade of JupyterLab v4 (when it will come).

To ease maintenance on the cookiecutter project, a solution may be to change the default branch to follow the one of the current major JupyterLab (and therefore drop master) - albeit this is unconventional.

Reply:
> To ease maintenance on the cookiecutter project, a solution may be to change the default branch to follow the one of the current major JupyterLab (and therefore drop master) - albeit this is unconventional.

Yes this is what the `jupyterlab_apod` repo does: https://github.com/jupyterlab/jupyterlab_apod